plugins/modem-manager/fu-mm-utils.c

```diff
@@ -46,6 +46,23 @@ fu_mm_utils_get_udev_port_info (GUdevDevice	 *device,
 		g_set_object (&parent, next);
 	}
 
+	if (!device_sysfs_path && g_strcmp0 (g_udev_device_get_subsystem (device), "wwan") == 0) {
```

Comment:
I added PCI support following the same logic as done in MM in this commit that I also commented about before: https://github.com/aleksander0m/fwupd/commit/abf7685e8e233be01da4f677508f88a690ff7ecf Could you see if that logic could fit better here? In that logic we look for the subsystem first with find_device_bus_subsystem() and then we apply different approaches to USB and PCI